scripts/helpers/index.js

```diff
@@ -9,3 +9,4 @@ export { selector } from './selector';
 export { checkAttr } from './check-attr';
 export { elementChildrenHeight } from './element-children-height';
 export { camelize } from './camelize';
+export { pasteInto } from './paste-handler';
```

Comment:
I think that this should go to https://github.com/infinum/eightshift-frontend-libs/tree/develop/scripts/editor because it is only editor related helper